### ISSUE TITLE
About dataset partition

### ISSUE THREAD

**Issue by ys10** [NVIDIA/waveglow#90]
Hi guys, I have trained a waveglow model on LJSpeech dataset.
Before starting training, I divide LJSpeech dataset into test set(LJ001-0001.wav - LJ001-0050.wav, 50 audios in total) and training set(other audios in LJSpeech dataset except audios in test set).
After training, I synthesized some samples by ground truth mels of LJSpeech dataset.
Samples whose ground truth mels in test set sound not good enough, compared to samples whose ground truth mels in training set.
I have checked your published samples(https://nv-adlr.github.io/WaveGlow), the quality of my samples refer to training set mels is close to quality of your samples, so I wonder wether ground truth mels of your published samples is in your training set?
BTW, could you please explain how you divide the LJSpeech dataset？

**Comment by ys10** [comment]
I noticed that the MOS of waveglow is 3.96 in your paper, I also wonder samples used in MOS-test were all refer to your test set? (BTW, I have completed a MOS-test, waveglow got 3.76 and recording got 4.55)

**Comment by ys10** [comment]
@rafaelvalle

**Comment by rafaelvalle** [comment]
No. There are no training set mel samples on the website. There are only test set samples.
We used about 100 samples for validation and another 100 for the test set.

When doing MOS experiments, it is necessary that each grader gets exposed to only a single file. Otherwise, there can be anchoring effects.

**Comment by cduguet** [comment]
I don't see any validation set being used in waveglow's code. Could you point me out if/where you used the validation dataset?
